### PR TITLE
sssctl: cache: remove, list and expire

### PR DESCRIPTION
**Pull request by patriki01**
Methods for removing, making cache expired and listing cache files.

### CODE REVIEW

File: sssd_test_framework/utils/sssctl.py [hunk 1]
```diff
@@ -90,3 +93,28 @@ def passkey_register(
             )
 
         return result.stdout_lines[-1].strip()
+
+    def cache_expire(self, *args: str) -> SSHProcessResult:
```
Comment (pbrezina) [inline sssd_test_framework/utils/sssctl.py:97]: I just added this in another pull request as I needed it to test a netgroup bug. Please, remove this one.

File: sssd_test_framework/utils/sssctl.py [hunk 2]
```diff
@@ -90,3 +93,28 @@ def passkey_register(
...
+        """
+        Run ``sssctl cache-expire`` command.
+
+        :param args: Additional arguments.
+        :type args: str
+        """
+        return self.host.ssh.exec(["sssctl", "cache-expire", *args])
+
+    def cache_ls(self) -> SSHProcessResult:
```
Comment (pbrezina) [inline sssd_test_framework/utils/sssctl.py:106]: Why is this needed?

This is not part of sssctl command so it should not be here.

File: sssd_test_framework/utils/sssctl.py [hunk 3]
```diff
@@ -90,3 +93,28 @@ def passkey_register(
...
+        """
+        Run ``ls 'cache_path'`` command.
+        Result output of ``ls`` is stored in result.stdout
+        """
+        return self.host.ssh.exec(["ls", self.cache_path])
+
+    def remove_cache_files(self) -> SSHProcessResult:
```
Comment (pbrezina) [inline sssd_test_framework/utils/sssctl.py:113]: Why is this needed?

This is not part of sssctl command so it should not be here.

If anything, there is `cache-remove`, but I'm not sure from the top of my head if it removes memory cache.

**Review by pbrezina — CHANGES_REQUESTED** [review]

File: sssd_test_framework/utils/sssctl.py (hunk 2 above)
Comment (patriki01) [reply sssd_test_framework/utils/sssctl.py:106]: It is probably not needed and even if, it shouldnt be there.
